### PR TITLE
Fix image percentage width/height in percentage w/h container

### LINKED ISSUE #2100: BUG img with 100% width inside a table cell will result in very small image
Hi
```
<table width="100%">
<tr>
     <td width="100%" style="border-spacing:0px;border:none;">
	<img alt="banner" src="https://dummyimage.com/600x400/000/fff" width="100%"  />
     </td>
</tr>
</table>
```
Tried using the newest version 0.8.5
will result in a image that is very small...

### PR DESCRIPTION
**Pull request by simonberger**
Fixes #2100

### CODE REVIEW

**Comment by simonberger** [comment]
Thanks. I'll add some test for this method text next days and then merge it.

**Comment by bsweeney** [comment]
Sounds good. I'm glad you're actively adding tests. I need to do better in that regard.

**Comment by simonberger** [comment]
Tests added. Found and fixed a bug because of it.
I had a wrong expectation of how length_in_pt works with a ref. I think it now works correctly in all cases. I merge it tomorrow.

File: src/FrameReflower/Image.php [hunk 1]
```diff
@@ -149,7 +115,7 @@ function get_min_max_width()
             $style->max_height !== "none"
         ) {
 
-            list( /*$x*/, /*$y*/, $w, $h) = $this->_frame->get_containing_block();
+            list($w, $h) = $this->_frame->get_containing_block();
```
[the remark this answers is not in the document]
Comment (bsweeney) [reply src/FrameReflower/Image.php:118]: ya know what, I'm going to comment on #2127.

**Pull request merged**